### DISCUSSION TITLE
files are missing  when I clone and ran the project

### DISCUSSION THREAD

**Discussion by thagalapallisaiprasad** [arafaysaleem/ez_tickets_app#101] [General]
Below is the screenshot I'm attaching for the errors. Most of them are files missing only.

<img width="397" alt="image" src="https://user-images.githubusercontent.com/29301440/156897474-b9c74ebc-1540-4043-8977-5f4c22cb36f7.png">
![WhatsApp Image 2022-03-06 at 00 53 33](https://user-images.githubusercontent.com/29301440/156897515-6d94cb26-7587-42d0-ba54-10753797e353.jpeg)
![WhatsApp Image 2022-03-06 at 00 55 44](https://user-images.githubusercontent.com/29301440/156897518-93ef8fa5-0c45-4197-8efe-abe4b48796cc.jpeg)

**Comment by arafaysaleem** [comment]
Ahh yes all of these are codegen files and not committed to vcs. You have to generate them on your machine using the build_runner

```dart
flutter pub run build_runner build --delete-conflicting-outputs
```

I should probably include this information in my README.
